common/model_package.go

```diff
@@ -326,3 +326,47 @@ func scanPackageModels(query *gocql.Query) ([]*PackageModel, error) {
 
 	return packageModels, nil
 }
+
+// TODO make query implicit here
+func QueryAllPackageModels(session *gocql.Session) ([]*PackageModel, error) {
+	var (
+		err        error
+		scanError  error
+		closeError error
+
+		author      string
+		repo        string
+		awesome_go  bool
+		description string
+		exists      bool
+		godoc_url   string
+		index_time  time.Time
+		search_blob string
+		versions    []string
+
+		query         = session.Query(`SELECT * FROM packages`)
+		iter          = query.Iter()
+		packageModels = make([]*PackageModel, 0)
+	)
+
+	for iter.Scan(&author, &repo, &awesome_go, &description, &exists, &godoc_url, &index_time, &search_blob, &versions) {
```

Comment:
Also is this really the only way to do this? I was a bit shocked we couldn't scan the fields directly into a related struct? If we leave it this way and changes we make to Package Schema has to be refactor, instead of through a single struct of some kind.